conjur-oss/values.schema.json

```diff
@@ -0,0 +1,234 @@
+            "cert": {
+              "type": "string"
+            },
+            "key": {
+              "type": "string",
+              "minLength": 1
+            },
+            "expiration": {
+              "type": "integer",
+              "minimum": 1
+            }
+          }
+        },
+        "url": {
+          "type": "string"
+        }
+      }
+    },
+    "dataKey": {
+      "type": "string",
+      "minLength": 1
+    },
+    "deployment": {
+      "properties": {
+        "annotations": {
+          "type": "object"
+        }
+      }
+    },
+    "image": {
+      "properties": {
+        "pullPolicy": {
+          "type": "string"
+        },
+        "repository": {
+          "type": "string"
+        },
+        "tag": {
+          "type": "number"
+        }
+      }
+    },
+    "nginx": {
+      "properties": {
+        "image": {
+          "properties": {
+            "pullPolicy": {
+              "type": "string"
+            },
+            "repository": {
+              "type": "string"
+            },
+            "tag": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "nodeSelector": {
+      "type": "object"
+    },
+    "postgres": {
+      "properties": {
+        "image": {
+          "properties": {
+            "pullPolicy": {
+              "type": "string"
+            },
+            "repository": {
+              "type": "string"
+            },
+            "tag": {
+              "type": "string"
+            }
+          }
+        },
+        "persistentVolume": {
+          "properties": {
+            "create": {
+              "type": "boolean"
+            },
+            "size": {
+              "type": "string"
+            },
+            "storageClass": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "postgresLabels": {
+      "type": "object"
+    },
+    "rbac": {
+      "properties": {
+        "create": {
+          "type": "boolean"
+        }
+      }
+    },
+    "replicaCount": {
+      "type": "integer",
+      "minimum": 1
+    },
+    "resources": {
+      "type": "object"
+    },
+    "service": {
+      "properties": {
+        "external": {
+          "properties": {
+            "annotations": {
+              "type": "object"
+            },
+            "enabled": {
+              "type": "boolean"
+            },
+            "port": {
+              "type": "integer"
+            }
+          }
+        },
+        "internal": {
+          "properties": {
+            "annotations": {
+              "type": "object"
+            },
+            "port": {
+              "type": "integer"
+            },
+            "type": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "serviceAccount": {
+      "properties": {
+        "create": {
+          "type": "boolean"
+        },
+        "name": {
+          "type": "string"
```

Comment:
For some reason, declaring "name" field as type "string" causes an error when "string" isn't supplied. I think the fix is to comment out the "name" field in values.yaml, e.g. replace this:
```
  name:
```
with:
```
  # name:
```